Documentation/dev-guide/grpc_naming.md

```diff
@@ -8,8 +8,8 @@ The etcd client provides a gRPC resolver for resolving gRPC endpoints with an et
 
 ```go
 import (
-	"github.com/coreos/etcd/clientv3"
-	etcdnaming "github.com/coreos/etcd/clientv3/naming"
+	"github.com/etcd-io/etcd/clientv3"
+	etcdnaming "github.com/etcd-io/etcd/clientv3/naming"
 
 	"google.golang.org/grpc"
 )
```

Comment:
Hi @vimalk78 import will now be `go.etcd.io` like https://github.com/etcd-io/etcd/commit/fced933294e86899842be026058a94b9baeefc97

Reply:
thanks @hexfusion , missed it. done !